main.tf

```diff
@@ -8,7 +8,8 @@ locals {
 }
 
 module "label" {
-  source = "git::https://github.com/cloudposse/terraform-null-label.git?ref=tags/0.19.2"
+  source  = "cloudposse/label/null"
+  version = "0.19.2"
```

Comment:
```suggestion
  version = "0.22.0"
```

Reply:
Yes, just wanted to show that it was not auto-updated.